src/main/resources/hudson/plugins/emailext/ExtendedEmailPublisher/config.groovy

```diff
@@ -72,9 +73,9 @@ f.advanced(title: _("Advanced Settings")) {
     f.checkbox(name: "project_save_output", checked: instance?.saveOutput)
   }
 
-  def configuredTriggers = instance != null ? instance.configuredTriggers : [hudson.plugins.emailext.plugins.trigger.FailureTrigger.createDefault()]
+  def configuredTriggers = instance != null ? instance.configuredTriggers : [hudson.plugins.emailext.plugins.trigger.FailureTrigger.createDefault(),hudson.plugins.emailext.plugins.trigger.UnstableTrigger.createDefault(),hudson.plugins.emailext.plugins.trigger.StatusChangedTrigger.createDefault()]
```

Comment:
Ok, I stand corrected. I would rather have it implemented so that people can choose the defaults in the global config then.